package.json

```diff
@@ -83,7 +83,7 @@
         "build:storybook": "cd packages/components && build-storybook --output-dir .out",
         "build:gh-pages": "f () { nx run-many --target=build projects=@deriv/components,@deriv/p2p && nx run-many --target=build --projects=@deriv/cashier,@deriv/account,@deriv/cfd,@deriv/reports && npm run build:local $1 ;}; f",
         "clean": "echo \"Remove $(git rev-parse --show-toplevel)/node_modules\" && lerna clean && rm -rf \"$(git rev-parse --show-toplevel)/node_modules\"",
-        "deploy": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy $@ ;}; f",
+        "deploy": "f () { export NODE_ENV=staging && npm run build:all && lerna exec --scope @deriv/core -- npm run deploy $@ && export NODE_ENV= ;}; f",
```

Comment:
Why do we need this change?

Reply:
That command was having issue with deploying on github pages. @balakrishna-deriv changed it to match what he has in his package.json I believe.

Reply:
The old command doesn't work properly after the NX migration. So, we had to update the command.